src/WalkingModule/app/robots/ergoCubGazeboV1/dcm_walking_with_joypad.ini

```diff
@@ -30,6 +30,9 @@ maximum_local_zmp                   (0.4, 0.3)
 # If set to true, the desired ZMP is used directly in the COM/ZMP controller
 skip_dcm_controller                  0
 
+# If set true, we remove the zmp-com offset at startworking and rotate in evaluateZMP
+remove_zmp_offset                   0
+
```

Comment:
Can you add this to all the robot files? It is a little tedious, but at least we know that we have all the parameters ready in all the configuration files

Reply:
Working on this :)

Reply:
Done in 099939671b8c10c9a272b031cc4ee938ec9da417